lib/streamlit/elements/vega_charts.py

```diff
@@ -1010,6 +1011,11 @@ def bar_chart(
               as the number of y values (e.g. ``color=["#fd0", "#f0f", "#04f"]``
               for three lines).
 
+        horizontal : bool
+            Determines the orientation of the chart:
+            * True: Displays the chart horizontally, with the x-axis and y-axis swapped.
+            * False: Displays the chart vertically (default).
```

Comment:
nit: I think our docstring convention for boolean flags is not to use bullets, but @sfc-gh-dmatthews is probably doing another iteration on that before release